references/kvm-admin-api/test/integration/features/kvm-admin.feature

```diff
@@ -65,6 +65,7 @@ Feature:
     Then response code should be 403
 
   Scenario: Retrieve an entry after deleting it
+    Given I waited for 2 seconds
```

Comment:
trivial: gherkin steps should be in the present tense e.g. `Given I wait for 2 seconds`